test/cli.test.ts

```diff
@@ -34,14 +37,11 @@ enum Status {
 async function getStatus(
   asyncFn: () => Promise<unknown>
 ): Promise<typeof Status['Success'] | typeof Status['Error']> {
-  let error = undefined;
-  try {
-    await asyncFn();
-  } catch (err) {
-    error = err;
```

Comment:
We'll need to revert this part after applying the suggestion of throwing an error in `runFiles`